README.rst

```diff
@@ -26,5 +26,5 @@ follows::
   $ virtualenv -p YOUR_PYTHON env
   $ source env/bin/activate
   $ pip install -e .
-  $ pip install pytest
+  $ pip install pytest mock
```

Comment:
use test requirements and change here w/ `pip install -e .[test]`
https://stackoverflow.com/questions/29870629/pip-install-test-dependencies-for-tox-from-setup-py